### ISSUE TITLE
On some mov files a preview is not generated and QuickLook not work

### ISSUE THREAD

**Issue by exeLexx** [Marginal/QLVideo#40]
Example: Apple Animation, 1920 × 1080, 25 fps, 4 sec duration.
```
iMac:~ Stripedberry$ qlmanage -m
server: living for 2361s (921 requests handled) - instant off: yes - arch: X86_64 - user id: 501
memory used: 2 MB (2137072 bytes) - used descriptors: 21/2560
last burst: during 1.572s - 3 requests - 0.141s idle
plugins:
  org.smpte.mxf -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  com.microsoft.ico -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  org.openxmlformats.presentationml.slideshow -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.adobe.pdf -> /System/Library/QuickLook/PDF.qlgenerator (720.7)
  com.apple.localized-pdf-bundle -> /System/Library/QuickLook/LocPDF.qlgenerator (720.7)
  public.3d-content -> /System/Library/QuickLook/SceneKit.qlgenerator (386.13)
  com.apple.iwork.keynote.key -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.ical.bookmark -> /System/Library/QuickLook/iCal.qlgenerator (720.7)
  com.apple.application -> /System/Library/QuickLook/StandardBundles.qlgenerator (720.7)
  com.microsoft.word.doc -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.microsoft.powerpoint.pot -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  org.openxmlformats.wordprocessingml.template.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.iwork.keynote.sffkey -> /Library/QuickLook/iWork.qlgenerator (166)
  dyn.ah62d4qmuhk2x43dts71g255bqu -> /System/Library/QuickLook/Package.qlgenerator (720.7 - loaded)
  com.apple.ichat.ichat -> /System/Library/QuickLook/iChat.qlgenerator (5500)
  com.apple.notes.externalrecord -> /System/Library/QuickLook/Notes.qlgenerator (719.68)
  com.microsoft.word.dot -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.iwork.keynote.sffkth -> /Library/QuickLook/iWork.qlgenerator (166)
  org.openxmlformats.presentationml.template.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.compuserve.gif -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  com.bittorrent.torrent -> /Applications/Transmission.app/Contents/Library/QuickLook/QuickLookPlugin.qlgenerator (1.0)
  public.jpeg -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  com.apple.keynote.key -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.mail.email -> /System/Library/QuickLook/Mail.qlgenerator (3273)
  com.apple.iwork.pages.sfftemplate -> /Library/QuickLook/iWork.qlgenerator (166)
  com.parallels.vm.vmpackage -> /Applications/Parallels Desktop.app/Contents/Library/QuickLook/ParallelsQL.qlgenerator (10.0.0)
  public.movie -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  com.apple.iwork.keynote.kth -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.iwork.numbers.sffnumbers -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.package -> /System/Library/QuickLook/Package.qlgenerator (720.7 - loaded)
  org.openxmlformats.wordprocessingml.document.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.iwork.pages.sffpages -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.ical.ics -> /System/Library/QuickLook/iCal.qlgenerator (720.7)
  org.oasis-open.opendocument.text -> /System/Library/QuickLook/Text.qlgenerator (720.7)
  public.mp3 -> /System/Library/QuickLook/Audio.qlgenerator (720.7 - loaded)
  org.openxmlformats.wordprocessingml.template -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.ical.ics.event -> /System/Library/QuickLook/iCal.qlgenerator (720.7)
  public.png -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  com.microsoft.excel.xlt -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.iwork.numbers.sfftemplate -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.iwork.keynote.key-tef -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.ibooksauthor.book -> /Library/QuickLook/iBooksAuthor.qlgenerator (1.0)
  com.apple.internet-location -> /System/Library/QuickLook/Clippings.qlgenerator (720.7)
  com.rsa.pkcs-12 -> /System/Library/QuickLook/Security.qlgenerator (55079.50.4)
  org.openxmlformats.presentationml.slideshow.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.microsoft.word.stationery -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.ibooks-folder -> /System/Library/QuickLook/iBooks.qlgenerator (1)
  public.x509-printable-encoded-archive -> /System/Library/QuickLook/Security.qlgenerator (55079.50.4)
  public.xml -> /System/Library/QuickLook/Text.qlgenerator (720.7)
  org.openxmlformats.wordprocessingml.document -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.rtfd -> /System/Library/QuickLook/Text.qlgenerator (720.7)
  com.parallels.vm.vmpackage.compressed -> /Applications/Parallels Desktop.app/Contents/Library/QuickLook/ParallelsQL.qlgenerator (10.0.0)
  com.apple.systempreference.prefpane -> /System/Library/QuickLook/StandardBundles.qlgenerator (720.7)
  org.openxmlformats.spreadsheetml.template -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.keychain -> /System/Library/QuickLook/Security.qlgenerator (55079.50.4)
  public.font -> /System/Library/QuickLook/Font.qlgenerator (720.7)
  public.x509-certificate -> /System/Library/QuickLook/Security.qlgenerator (55079.50.4)
  com.apple.iwork.pages.template -> /Library/QuickLook/iWork.qlgenerator (166)
  com.adobe.illustrator.ai-image -> /System/Library/QuickLook/EPS.qlgenerator (720.7)
  public.mpeg-4 -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  public.vcard -> /System/Library/QuickLook/Contact.qlgenerator (720.7)
  com.apple.finder.textclipping -> /System/Library/QuickLook/Clippings.qlgenerator (720.7)
  org.openxmlformats.presentationml.template -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.adobe.photoshop-image -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  com.apple.iwork.numbers.numbers -> /Library/QuickLook/iWork.qlgenerator (166)
  org.openxmlformats.spreadsheetml.template.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.iwork.pages.pages -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.ichat.transcript -> /System/Library/QuickLook/iChat.qlgenerator (5500)
  com.apple.mail.emlx -> /System/Library/QuickLook/Mail.qlgenerator (3273)
  com.apple.iconset -> /System/Library/QuickLook/Icon.qlgenerator (720.7)
  com.apple.addressbook.group -> /System/Library/QuickLook/Contact.qlgenerator (720.7)
  com.apple.property-list -> /System/Library/QuickLook/Text.qlgenerator (720.7)
  org.idpf.epub-container -> /System/Library/QuickLook/iBooks.qlgenerator (1)
  com.apple.addressbook.person -> /System/Library/QuickLook/Contact.qlgenerator (720.7)
  com.apple.safari.history -> /System/Library/QuickLook/Bookmark.qlgenerator (720.7)
  public.html -> /System/Library/QuickLook/Web.qlgenerator (720.7)
  com.microsoft.waveform-audio -> /System/Library/QuickLook/Audio.qlgenerator (720.7 - loaded)
  com.apple.quicktime-movie -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  com.apple.iwork.numbers.template -> /Library/QuickLook/iWork.qlgenerator (166)
  com.microsoft.powerpoint.pps -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  public.plain-text -> /Applications/TextMate.app/Contents/Library/QuickLook/TextMateQL.qlgenerator (1.0)
  public.source-code -> /Applications/TextMate.app/Contents/Library/QuickLook/TextMateQL.qlgenerator (1.0)
  com.apple.safari.bookmark -> /System/Library/QuickLook/Bookmark.qlgenerator (720.7)
  public.audio -> /System/Library/QuickLook/Audio.qlgenerator (720.7 - loaded)
  org.idpf.epub-folder -> /System/Library/QuickLook/iBooks.qlgenerator (1)
  com.apple.pict -> /System/Library/QuickLook/Clippings.qlgenerator (720.7)
  public.xhtml -> /System/Library/QuickLook/Web.qlgenerator (720.7)
  org.openxmlformats.presentationml.presentation.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.finder.pictclipping -> /System/Library/QuickLook/Clippings.qlgenerator (720.7)
  com.apple.ical.bookmark.todo -> /System/Library/QuickLook/iCal.qlgenerator (720.7)
  com.apple.iwork.numbers.numbers-tef -> /Library/QuickLook/iWork.qlgenerator (166)
  com.apple.webarchive -> /System/Library/QuickLook/Web.qlgenerator (720.7)
  org.openxmlformats.presentationml.presentation -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  public.svg-image -> /System/Library/QuickLook/Web.qlgenerator (720.7)
  com.apple.private.live-photo-bundle -> /System/Library/QuickLook/LivePhoto.qlgenerator (720.7)
  public.text -> /Applications/TextMate.app/Contents/Library/QuickLook/TextMateQL.qlgenerator (1.0)
  com.apple.m4v-video -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  public.image -> /System/Library/QuickLook/Image.qlgenerator (720.7 - loaded)
  org.openxmlformats.spreadsheetml.sheet -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.microsoft.excel.xls -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  org.openxmlformats.spreadsheetml.sheet.macroenabled -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  public.rtf -> /System/Library/QuickLook/Text.qlgenerator (720.7)
  net.maxon.cinema4d.c4d -> /Applications/Adobe After Effects CC 2017/Plug-ins/MAXON CINEWARE AE/(CINEWARE Support)/lite/CINEMA 4D Lite.app/Contents/Library/QuickLook/CINEMA 4D.qlgenerator (1.1)
  org.bittorrent.torrent -> /Applications/Transmission.app/Contents/Library/QuickLook/QuickLookPlugin.qlgenerator (1.0)
  com.apple.iwork.pages.pages-tef -> /Library/QuickLook/iWork.qlgenerator (166)
  com.microsoft.powerpoint.ppt -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  public.comma-separated-values-text -> /System/Library/QuickLook/Office.qlgenerator (32 - loaded)
  com.apple.ical.ics.todo -> /System/Library/QuickLook/iCal.qlgenerator (720.7)
  com.adobe.encapsulated-postscript -> /System/Library/QuickLook/EPS.qlgenerator (720.7)
  com.microsoft.advanced-systems-format -> /Library/QuickLook/Video.qlgenerator (1.8.7 - loaded)
  com.microsoft.windows-executable -> /Applications/Parallels Desktop.app/Contents/Library/QuickLook/ExeQL.qlgenerator (12.1.3 - loaded)

iMac:~ Stripedberry$ qlmanage -p -d1 /Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov
Testing Quick Look preview with files:
	/Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov
[DEBUG] Preview test for file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov. Content type UTI: com.apple.quicktime-movie
[DEBUG] Previewing file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov. Content type UTI: com.apple.quicktime-movie. Generator used: <QLGenerator /Library/QuickLook/Video.qlgenerator>
[DEBUG] Loading <QLGenerator /Library/QuickLook/Video.qlgenerator>
[DEBUG] Previewing file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov did not produce anything
[DEBUG] Thumbnailing file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov - type: com.apple.quicktime-movie - generator: <QLGenerator /Library/QuickLook/Video.qlgenerator> (try #1)
[DEBUG] Thumbnailing file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov. Content type UTI: com.apple.quicktime-movie. Generator used: <QLGenerator /Library/QuickLook/Video.qlgenerator>
[DEBUG] Thumbnailing file:///Users/Stripedberry/Desktop/20_KB_INVADER_2D_BROOKLYN_SAUSAGE_PAL.mov did not produce anything
2017-05-18 18:08:12.538 qlmanage[2354:77619] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0xaa03, name = 'com.apple.coredrag'
See /usr/include/servers/bootstrap_defs.h for the error codes.
2017-05-18 18:08:12.728 qlmanage[2354:77619] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0xd73b, name = 'com.apple.tsm.portname'
See /usr/include/servers/bootstrap_defs.h for the error codes.
2017-05-18 18:08:12.729 qlmanage[2354:77619] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0xda03, name = 'com.apple.CFPasteboardClient'
See /usr/include/servers/bootstrap_defs.h for the error codes.
```

But on other files like this (18 of 23) QL work.

System Version:	macOS 10.12.4 (16E195)
QLVideo Release 1.87

**Comment by Marginal** [comment]
Thanks for the report. Can you make this file available to me for testing?

**Comment by exeLexx** [comment]
Of course! https://yadi.sk/i/JagRhc-t3JJiq8

**Comment by Marginal** [comment]
The plugin kept looking for a lighter frame, didn't find one, so gave up.
Will be fixed in the next release in a week or two.

**Comment by exeLexx** [comment]
👍🏻 Thanks you for your work!

30 мая 2017 г., 3:44 +0300, Jonathan Harris <notifications@github.com>, писал:

**Comment by Marginal** [comment]
Fixed in [Release 1.88](https://github.com/Marginal/QLVideo/releases/latest).
